### ISSUE TITLE
Use SVG or higher resolution badge F-Droid

### ISSUE THREAD

**Issue by PanderMusubi** [binwiederhier/ntfy#1170]
:bulb: **Idea**
Use SVG or hIgher resolution badge F-Droid

:computer: **Target components**
Website

PS Consider adding the three badges also to the README.

**Comment by PanderMusubi** [comment]
![Screenshot_20240823-125652_Fennec_1_1](https://github.com/user-attachments/assets/7b123e38-adb9-4880-903c-a6cc5042d4fb)

**Comment by Unbloated** [comment]
If SVG is available, always use it!

**Comment by PanderMusubi** [comment]
yes, see e.g. https://github.com/f-droid/artwork/blob/master/badge/get-it-on.svg

**Comment by binwiederhier** [comment]
Done. Related to https://github.com/binwiederhier/ntfy/commit/6978fa69a8f7368f1ab86da73f6919c66de3cf72 though the commit on the website is private since that's not open source.
